test/e2e/tests/test_model_package.py

```diff
@@ -0,0 +1,266 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Integration tests for the SageMaker modelPackage API.
+"""
+
+import botocore
+import pytest
+import logging
+from typing import Dict
+
+from acktest.resources import random_suffix_name
+from acktest.k8s import resource as k8s
+from e2e import (
+    service_marker,
+    wait_for_status,
+    create_sagemaker_resource,
+    sagemaker_client,
+    assert_tags_in_sync,
+)
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.bootstrap_resources import get_bootstrap_resources
+from e2e.common import config as cfg
+
+RESOURCE_PLURAL = "modelpackages"
+
+
+@pytest.fixture(scope="function")
+def xgboost_model_package_group():
+    resource_name = random_suffix_name("xgboost-model-package-group", 32)
+
+    replacements = REPLACEMENT_VALUES.copy()
+    replacements["MODEL_PACKAGE_GROUP_NAME"] = resource_name
+
+    (
+        model_package_group_reference,
+        model_package_group_spec,
+        model_package_group_resource,
+    ) = create_sagemaker_resource(
+        resource_plural=cfg.MODEL_PACKAGE_GROUP_RESOURCE_PLURAL,
+        resource_name=resource_name,
+        spec_file="xgboost_model_package_group",
+        replacements=replacements,
+    )
+    assert model_package_group_resource is not None
+    if k8s.get_resource_arn(model_package_group_resource) is None:
+        logging.debug(
```

Comment:
nit: change to logging.error. Because of [this configuration](https://github.com/aws-controllers-k8s/sagemaker-controller/blob/522c147abcca72d97539f7296f000105b7b1dc26/test/e2e/service_bootstrap.py#L123), debug logs are not getting printed

we need to change for others as well but lets not do in this PR